2-mandatory/2-financial-times.js

```diff
@@ -23,59 +44,74 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  newArticle = [];
+  for (const element of allArticleTitles) {
+    if (/\d/.test(element)) {
+      newArticle.push(element);
+    }
+  }
+  return newArticle;
 }
 
 /*
     The Financial Times wants to understand what the average number of characters in an article title is.
     Implement the function below to return this number - rounded to the nearest integer.
 */
 function averageNumberOfCharacters(allArticleTitles) {
-    // TODO
+  let sum = 0;
+  // let result=0;
+  for (const i of allArticleTitles) {
+    sum = sum + i.length;
```

Comment:
you found the easy and short way .very good.

Reply:
Thank you Shadi :)